{{cookiecutter.repo_name}}/.github/workflows/{{cookiecutter._ci_name}}.yaml

```diff
@@ -171,7 +171,7 @@ jobs:
 
     - name: Check package build
       run: |
-        DISTRIBUTION=$(ls -t1 dist/{{ cookiecutter.repo_name }}-*.tar.gz | head -n 1)
+        DISTRIBUTION=$(ls -t1 dist/{{ cookiecutter.repo_name.lower().replace('-', '_') }}-*.tar.gz | head -n 1)
```

Comment:
Ah sorry if I'm too late, didn't know if you wanted comments - I was wondering if "."  also needed to be accounted for as per https://peps.python.org/pep-0503/#normalized-names

Reply:
Ah that's a good idea, thanks for the comment -- I didn't request reviews since this seemed like a small change to me. I've raised https://github.com/MDAnalysis/cookiecutter-mdakit/issues/119 for this now